systempony

```diff
@@ -10,6 +10,18 @@ FSTYPE=
 # Interface to get the IP address from
 IP_INTERFACE="eth0"
 
+function human_readable() {
+    echo "$1" | awk '
+    function human(x) {
+        if (x<1000) {return x} else {x/=1024}
+        s="kMGTEPYZ";
+        while (x>=1000 && length(s)>1)
+            {x/=1024; s=substr(s,2)}
+        return int(x+0.5) substr(s,1,1)
```

Comment:
I think it would look better with `printf("%.1f", x)` than `int(x+0.5)`

Reply:
I copied this code from an SO answer.